ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -45,9 +53,9 @@ Diese Suchanfrage gibt alle Patienten zurück zum Client, welche in einem Adress
 
 ### Token
 
-Der Modifier `:not` MUSS für alle spezifizierten Suchparameter vom Typ 'Token' unterstützt werden, sofern diese auf die Datentypen "code", "Coding" oder "CodeableConcept" verwendet werden.
+Der Modifier `:not` KANN für alle spezifizierten Suchparameter vom Typ 'Token' unterstützt werden, sofern diese auf die Datentypen "code", "Coding" oder "CodeableConcept" verwendet werden.
```

Comment:
question: auch dieser Modifier wurde bisher verpflichtend getestet:

* insert searchOperation (Procedure, xml, ?_id:not={{procedure-read-id}}&status:not=completed)

soll er wirklich entfallen?

Reply:
@alexzautke und @simoneOnFhir könntet ihr das beantowrten?

Reply:
:not ist nicht spezifisch für Suchparameter vom Typ Reference. An den Allgemeinen Anforderungen oder denen anderer Suchparameter sollte nichts geändert werden.

Reply:
wenn ich es richtig sehe, bezieht sich die markierte Stelle bezieht sich auf Token-basierte Suchparameter allgemein. so wie es oben steht, wäre die Unterstützung von bspw. status:not=completed nun optional und nicht mehr verpflichtend wie früher